datarepo-clienttests/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
         findbugsAnnotations = "3.0.1"
 
         jackson = "2.10.2"
-        kubernetesClient = "8.0.2"
+        kubernetesClient = "10.0.0"
```

Comment:
We currently also have a reference to the Kuberenetes client in the main jade-data-repo gradle.  I'm not totally sure if it's needed there, so we should probably either get rid of the reference or also bump the version there.

Reply:
Good point. Done.